client/scripts/views/pages/notifications.ts

```diff
@@ -776,6 +788,7 @@ const NotificationsPage: m.Component<{}, {
               selectedCommunity.network === ChainNetwork.Kulupu && m(KulupuChainEventNotifications),
               selectedCommunity.network === ChainNetwork.Kusama && m(KusamaChainEventNotifications),
               selectedCommunity.network === ChainNetwork.Polkadot && m(PolkadotChainEventNotifications),
+              selectedCommunity.network === ChainNetwork.Aave && m(DydxChainEventNotifications),
```

Comment:
is network supposed to be Aave or Dydx here? Confused.

Reply:
Both dydx and aave use ChainNetwork.AAVE right now